tutorials/headless_rendering.md

```diff
@@ -0,0 +1,68 @@
+\page headless_rendering Headless Rendering 
+  
+It is often desirable to run simulation on a remote computer, such as
+a computer managed by cloud provider, in order to paralellize work or access
+specific compute resources. Simulated sensors that require GPU access has
+historically been difficult to use on a remote computer due to opengl's
+X server requirement. This issue can be resolved through installation and
+proper configuration of X, but the steps can be complex and error prone.
+
+An easier solution is through the use of [EGL](https://www.khronos.org/egl), which allows for the the creation of rendering surfaces without an X server. Ignition Gazebo has added support for EGL via the `--headless-rendering` command line option. For example:
+
+```
+ign gazebo -v 4 -s --headless-rendering sensors_demo.sdf
+```
+
+If you are using Ignition Gazebo as a library, then you can configure the
+server to use headless rendering through the
+`ServerConfig::SetHeadlessRendering(bool)` function.
+
+## AWS Example
+
+This example will guide you through the process of launching and
+configuring an AWS GPU instance with Gazebo running headless.
+
+1. Go to the [AWS EC2 service](https://console.aws.amazon.com/ec2)
+2. Click the `Launch Instance` button in the upper right.
+3. Select `Ubuntu Server` version 20.04 or greater from the AMI list.
+4. Choose a GPU enabled instance type, such as `g3.4xlarge`.
```

Comment:
Does it have to be a GPU instance? I understood one of the benefits of EGL was running on non-GPU instances without NVIDIA drivers.

Reply:
@iche033 , does lack of a GPU cause EGL to use software rendering?

Reply:
yeah here's [more info](https://www.ogre3d.org/2021/02/06/ogre-2-2-5-cerberus-released-and-egl-headless-support) in ogre-next's post under Headless SW Rasterization. There is software rasterization through Mesa.

Reply:
Added note about software rendering in [2f43048](https://github.com/ignitionrobotics/ign-gazebo/pull/1386/commits/2f4304881881d7c87103ee62420cf421be3ac9d4)